test/dotnet-add-reference.Tests/GivenDotnetAddReference.cs

```diff
@@ -114,8 +112,8 @@ public void WhenTooManyArgumentsArePassedItPrintsError()
                     .WithProject("one two three")
                     .Execute("proj.csproj");
             cmd.ExitCode.Should().NotBe(0);
-            cmd.StdErr.Should().Be("Unrecognized command or argument 'two'");
-            cmd.StdOut.Should().Be("Specify --help for a list of available options and commands.");
```

Comment:
Prior to this PR, we're inconsistent. In some cases when there's a parse error, we tell the user to request help. In others, we immediately show help. I've converged on the latter behavior but verifying it here is redundant with a number of other tests, and doing a text comparison to the complete help text is fairly high maintenance, so I opted to go to a single assertion here.